languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -9175,6 +9186,17 @@ though, and has since been slightly modified)-->
       </rule>
       <rulegroup id="D_N_QUE_R_AVOIR_PPA" name="Relative avec erreur d'accord" default="off">
           <rule name="f s">
+              <antipattern>
+                  <token postag="SENT_START"/>
+                  <token postag="D f s"/>
+                  <token min="0" max="1" postag="A"/>
+                  <token postag="N f s"/>
+                  <token regexp="yes">qu[e']</token>
```

Comment:
We might want to include the entire palette of possible apostrophe candidates: `[’´'‛′‘]`

Reply:
As it is now, `<token regexp="yes">qu[e']</token>` covers (internally) typewriter apostrophe (') and typographical apostrophe (’). The other characters are not tokenized this way. They are separate tokens. So, it is pointless to add them here. They are considered typographical errors by other rules.

Reply:
Okay, cool! Thank you for the info :)